frontends/web/src/routes/account/send/send.tsx

```diff
@@ -221,7 +197,7 @@ class Send extends Component<Props, State> {
   };
 
   private handleKeyDown = (e: KeyboardEvent) => {
-    if (e.keyCode === 27 && !this.state.activeCoinControl && !this.state.activeScanQR) {
+    if (e.keyCode === 27 && !this.state.activeCoinControl) {
```

Comment:
I tried combining ScanQRDialog and ReceiverAddressInput @thisconnect . Unfortunately this isn't 100% working "smoothly". See this bit of code that I'm highlighting.

We somehow still need the parent (send page) to know if Dialog is opened, so the firing of "escape" can work correctly in the send screen; which is "escape" should close qrdialog when qrdialog is opened, otherwise it closes the whole page. The current (broken) behaviour will close the whole page regardless of the dialog's condition.

To fix this, I'll put back `activeScanQR` in the parent for now and pass that state and its setter to `ReceiverAddressInput` - please lmk what you think though. I don't think this is a very clean fix... but a simple enough change for now.

This might be an indication that we may need to refactor/rethink the way our Dialog works.

Reply:
Another hacky fix is we can try to check if the actual dom is there e.g `document.querySelector()` - but I'd classify this as a hack.